project/src/main/webapp/app.module.js

```diff
@@ -0,0 +1,35 @@
+//Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// use modern JavaScript (ES5)
+"use strict"
+
+/**
+ * Adds a official election/candidate information (brief) to the homepage.
+ */
+function addOfficialElectionCandidateInformation() {
+  // Capture user input of location.
+
+  // Send GET request to /data with location.
+
+  // Unpack response.
+
+  // Add official election/candidate information to HTML.
+
+  // Add URLs, that send GET request to /candidate with candidate ID, to HTML.
+
+}
+
+angular.module('voterCentral',
+    ['navigation', 'directoryPage', 'candidatePage']);
```

Comment:
Have you decided whether to include the `eduationPage`?

Reply:
Either way, it shouldn't need a corresponding module, as it's static and doesn't need a component. Theoretically if I get the routing working in the future, I could turn it from a regular HTML file into a template. But it works as is.

Reply:
Sounds great!